packages/local_auth/local_auth_android/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.1
+
+* Fix getEnrolledBiometrics to be in line with iOS implementation:
```

Comment:
We shouldn't reference iOS here; the correlation with iOS is an internal implementation (which isn't even relevant any more, since we're no longer sharing the method channel). What's important is that it fixes the behavior of the platform implementation, which currently doesn't match the documented behavior.